make error, when generating doc/guide.pdf, LuaTeX

```
rustdoc: doc/guide.html
pandoc: doc/guide.tex
/usr/bin/pandoc --standalone --toc --number-sections --include-before-body=doc/version.tex --from=markdown --include-before-body=doc/footer.tex --to=latex /home/liigo/rust/rust/src/doc/guide.md --output=doc/guide.tex
latex compiler: doc/guide.pdf
This is LuaTeX, Version beta-0.70.2-2012070300 (TeX Live 2012/Debian)
 restricted \write18 enabled.
make: *** [doc/guide.pdf] Error 1
```

--- a/configure
+++ b/configure
@@ -653,9 +653,7 @@ probe CFG_GRUN             grun
 probe CFG_FLEX             flex
 probe CFG_BISON            bison
 probe CFG_PANDOC           pandoc
-probe CFG_PDFLATEX         pdflatex
 probe CFG_XELATEX          xelatex
-probe CFG_LUALATEX         lualatex
 probe CFG_GDB              gdb
 probe CFG_LLDB             lldb
 

--- a/mk/docs.mk
+++ b/mk/docs.mk
@@ -85,26 +85,15 @@ else
 HTML_DEPS :=
 endif
 
-# Check for the various external utilities for the EPUB/PDF docs:
+# Check for xelatex
 
-ifeq ($(CFG_LUALATEX),)
-  $(info cfg: no lualatex found, deferring to xelatex)
-  ifeq ($(CFG_XELATEX),)
-    $(info cfg: no xelatex found, deferring to pdflatex)
-    ifeq ($(CFG_PDFLATEX),)
-      $(info cfg: no pdflatex found, disabling LaTeX docs)
-      NO_PDF_DOCS = 1
-	else
-      CFG_LATEX := $(CFG_PDFLATEX)
-    endif
-  else
+ifeq ($(CFG_XELATEX),)
     CFG_LATEX := $(CFG_XELATEX)
     XELATEX = 1
-  endif
-else
-  CFG_LATEX := $(CFG_LUALATEX)
+  else
+    $(info cfg: no xelatex found, disabling LaTeX docs)
+    NO_PDF_DOCS = 1
 endif
-
 
 ifeq ($(CFG_PANDOC),)
 $(info cfg: no pandoc found, omitting PDF and EPUB docs)

